libecs/include/pivot/ecs/Components/Camera.hxx

```diff
@@ -54,22 +52,6 @@ public:
     /// Get camera view
     inline glm::mat4 getView() const { return glm::lookAt(position, position + front, up); }
 
-    /// Get CameraData of the camera
-    pivot::graphics::CameraData getGPUCameraData(float fFOV, float fAspectRatio,
-                                                 float fCloseClippingPlane = PIVOT_MIN_PROJECTION_LIMIT,
-                                                 float fFarClippingPlane = PIVOT_MAX_PROJECTION_LIMIT) const
-    {
-        auto projection = getProjection(fFOV, fAspectRatio, fCloseClippingPlane, fFarClippingPlane);
-        projection[1][1] *= -1;
-        auto view = getView();
-        return {
-            .position = position,
-            .view = view,
-            .projection = projection,
-            .viewProjection = projection * view,
-        };
-    }
-
```

Comment:
Isn't it already done? This function was moved in internal/camera and now takes a `Camera` object as first parameter

Reply:
I mean `getProjection`, `getView` etc...

Reply:
Oh the method**s** then 😆

Reply:
@zcorniere Do you think anything need to be done on this still ?

Reply:
@zcorniere Do you think anything need to be done on this still ?

Reply:
Well, my question still stand: is it possible to move the methods out of the Camera ? (`getProjection`, `getView`) It would allow the camera to be a regular component, whether we decide to add it into the ECS or not.